packages/contracts-bedrock/src/L2/L2ToL1MessagePasser.sol

```diff
@@ -101,4 +123,24 @@ contract L2ToL1MessagePasser is ISemver {
     function messageNonce() public view returns (uint256) {
         return Encoding.encodeVersionedNonce(msgNonce, MESSAGE_VERSION);
     }
+
+    /// @notice This function is used to enable the native deposit functionality
+    function enableNativeDeposit() external {
+        if (msg.sender != Constants.DEPOSITOR_ACCOUNT) {
+            revert("L2ToL1MessagePasser: Only the depositor can enable native deposits");
+        }
+        QKCConfigStorage storage $ = _getQKCConfigStorage();
+        $.disableNativeDeposit = false;
```

Comment:
I’m a bit confused, where is `$.disableNativeDeposit` used? Should we be checking it inside `initiateWithdrawal`?